### PR TITLE
Feature Spin-Orbit coupling

### PR DESCRIPTION
**Pull request by AndresOrtegaGuerrero**
This solves #689

### CODE REVIEW

**Comment by superstar54** [comment]
Hi @AndresOrtegaGuerrero, Thanks for the work! That's fast, and it's a very important feature. I will look at and test this PR.

File: src/aiidalab_qe/app/configuration/advanced.py [hunk 1]
```diff
@@ -209,6 +209,21 @@ def __init__(self, default_protocol=None, **kwargs):
             (self.etot_conv_thr, "disabled"),
             lambda override: not override,
         )
+        # Spin-Orbit calculation
+        self.spin_orbit = ipw.ToggleButtons(
+            options=[
+                ("W/o SOC", "wo_soc"),
+                ("SOC", "soc"),
```
Comment (superstar54) [inline src/aiidalab_qe/app/configuration/advanced.py:216]: I suggest using `On` and `Off` as options, just like `Magnetism.`

**Review by superstar54 — CHANGES_REQUESTED** [review]
Hi @AndresOrtegaGuerrero , thanks for the great work. Only a minor change is needed.

How would this PR affect the other plugins?

File: src/aiidalab_qe/common/setup_pseudos.py [hunk 2]
```diff
@@ -28,6 +28,10 @@
     f"PseudoDojo/{PSEUDODOJO_VERSION}/PBEsol/SR/standard/upf",
     f"PseudoDojo/{PSEUDODOJO_VERSION}/PBE/SR/stringent/upf",
     f"PseudoDojo/{PSEUDODOJO_VERSION}/PBEsol/SR/stringent/upf",
+    f"PseudoDojo/{PSEUDODOJO_VERSION}/PBE/FR/standard/upf",
+    f"PseudoDojo/{PSEUDODOJO_VERSION}/PBEsol/FR/standard/upf",
+    f"PseudoDojo/{PSEUDODOJO_VERSION}/PBE/FR/stringent/upf",
+    f"PseudoDojo/{PSEUDODOJO_VERSION}/PBEsol/FR/stringent/upf",
```
Comment (superstar54) [inline src/aiidalab_qe/common/setup_pseudos.py:34]: Does the soc always require the `FR` pseudos?

File: src/aiidalab_qe/app/configuration/advanced.py (hunk 1 above)
Comment (AndresOrtegaGuerrero) [reply src/aiidalab_qe/app/configuration/advanced.py:216]: @AndresOrtegaGuerrero  like options=[
                ("Off", "wo_soc"),
                ("on", "soc"), ?

Comment (superstar54) [reply src/aiidalab_qe/app/configuration/advanced.py:216]: Yes

**Review by superstar54 — APPROVED** [review]
LGTM!

**Pull request merged**